File: typescript/Two Pointers/pair_sum_sorted_brute_force.ts
function pairSumSortedBruteForce(nums: number[], target: number): number[] {
    const n = nums.length;
    for (let i = 0; i < n; i++) {
        for (let j = i + 1; j < n; j++) {
            if (nums[i] + nums[j] === target) 
                return [i, j];
        }
    }
    return [];
}

File: csharp/Two Pointers/PairSumSortedBruteForce.cs
public partial class Solution 
{
    public int[] PairSumSortedBruteForce(int[] nums, int target) 
    {
        int n = nums.Length;
        for (int i = 0; i < n; i++) 
        {
            for (int j = i + 1; j < n; j++) 
            {
                if (nums[i] + nums[j] == target) 
                {
                    return [i, j];
                }
            }
        }

        return [];
    }
}